src/app/build.gradle

```diff
@@ -98,7 +98,7 @@ task generateTestData() {
             commandLine 'docker', 'run', '--rm', '--entrypoint', 'create_orderly_demo.sh', '-u', uid, '-v', "$projectDir:/orderly", '-w', "/orderly", image, "."
         }
         
-        String owMigrateImage = "vimc/orderlyweb-migrate:mrc-1500"
+        String owMigrateImage = "vimc/orderlyweb-migrate:mrc-1573"
```

Comment:
Does this never get set to master?

Reply:
It does, after merge to master. We just hadn't got round to re-setting after mrc-1500 I guess!